.github/workflows/rust.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       matrix:
         directory: ${{ fromJSON(needs.find-crates.outputs.crates) }}
         exclude:
-          - directory: hoverkite-firmware
+          - directory: cross
```

Comment:
This job is now just being run under the root directory, and I guess that won't change, so you can remove the matrix entirely and simplify it a bit.

Reply:
Done myself.